cluster/gce/gci/configure-helper.sh

```diff
@@ -25,6 +25,21 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
+function convert-manifest-params {
+  # Convert the manifest's command's parameters.
+  # Previously the command is in the form of
+  # ["/bin/sh", "-c", "exec KUBE_EXEC_BINARY --param1=val1 --param2-val2"].
+  # Since we remove the bash dependency and the executable binary is called
+  # directly, the format should be changed to
+  # ["KUBE_EXEC_BINARY", "--param1=val1", "--param2-val2"].
```

Comment:
I think this comment is no longer accurate?

Reply:
haha, good catch!